simulation.sh

```diff
@@ -1,6 +1,7 @@
 package: simulation
 version: v1.0
 requires:
+  - VMC
```

Comment:
I think that this is not needed, as the dependence on VMC is already declared in GEANT3 and GEANT4_VMC

Reply:
It is not needed, just to make it explicit. But we can remove it of course

Reply:
indeed @ihrivnac should we make it explicit in the `fairroot.sh` recipe rather than here? I would just like to have it explicitly in central recipes so that it is clear what happens.

Reply:
I don't see a need for making it explicit here or in FairRoot, as the VMC is anyway nowhere else used (except for the AEGIS case where its usage is disputable)

Reply:
ROOT dependency can be dropped in FairRoot. We have also many explicit dependencies on ROOT in O2, and we do not add ROOT in requires in o2.sh as it is already implicitly included. Do you find this misleading as well?

Reply:
ok, true. If we remove ROOT from FairRoot then I agree
* remove ROOT from all VMC recipes
* remove ROOT from FairRoot
* remove VMC from simulation

I will test that and add it to the PR

Reply:
I have removed the requirements as listed in my previous comment

Reply:
Great, thank you